fuse_ll/src/fuse/mount.rs

```diff
@@ -148,13 +157,12 @@ mod param {
                 kernel_opts: None,
                 max_read: 0,
             };
-            options.iter().for_each(|&op| {
-                let mount_options = get_mount_options();
-                let option = mount_options
-                    .iter()
-                    .find(|x| x.regex.is_match(&op))
-                    .unwrap();
-                (option.parser)(&mut args, &option, &op)
+            let mount_options_map = super::get_mount_options_map();
+            options.iter().for_each(|op| {
+                let key = op.split('=').collect::<Vec<_>>()[0].to_string();
+                if let Some(option) = mount_options_map.get(&key) {
```

Comment:
Will we report error when the option is not defined?

Reply:
All the options that reach here will be valid options.
Option validation will be done by command line parser.

Reply:
If it's garanteed to be validate, `unwrap` is a clear way I think.

Reply:
Modified. Thanks.